spring-boot-starters/spring-boot-starter-data-geode/pom.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<groupId>org.springframework.boot</groupId>
+		<artifactId>spring-boot-starters</artifactId>
+		<version>1.4.0.BUILD-SNAPSHOT</version>
+	</parent>
+	<artifactId>spring-boot-starter-data-geode</artifactId>
+	<name>Spring Boot Data Geode Starter</name>
+	<description>
+		The Spring Boot Data Geode Starter including Spring Data GemFire with support for Apache Geode.
+	</description>
+	<url>http://projects.spring.io/spring-boot/</url>
+	<organization>
+		<name>Pivotal Software, Inc.</name>
+		<url>http://www.spring.io</url>
+	</organization>
+	<properties>
+		<main.basedir>${basedir}/../..</main.basedir>
+	</properties>
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.data</groupId>
+			<artifactId>spring-data-gemfire</artifactId>
+			<version>${spring-data-geode.version}</version>
```

Comment:
Also, _Spring Boot_ is currently limited in that a user (at her own risk) using a starter, or inheriting from the `spring-boot-starter-parent`, **cannot** override any particular _Spring Data_ module (e.g. _SD GemFire_, _SD Redis_, or otherwise) in a given _Spring Boot_ release (e.g. `1.3.3.RELEASE`) at all.  For instance...

// application pom.xml

<parent>
  <groupId>org.springframework.boot</groupId>
  <artifactId>spring-boot-starter-parent</artifactId>
  <version>1.4.0.BUILD-SNAPSHOT</version>
</parent>
...
<properties>
  <spring-data-gemfire.version>1.7.4.RELEASE</spring-data-gemfire.version>
  <spring-data-redis.version>1.6.4.RELEASE</spring-data-redis.version>
</properties>

<dependencies>
  <dependency>
    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-starter-data-gemfire</artifactId>
  </dependency>
  <dependency>
    <groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-starter-data-redis</artifactId>
  </dependency>
</dependencies>

Essentially, there is no reason to use the _Spring Boot_ Starters for either _GemFire_ or _Redis_ at this point since there is no way for the user to override the dependency version.  The user is effectively stuck with the `Hopper` versions, where as other [dependencies](https://github.com/spring-projects/spring-boot/blob/v1.4.0.M1/spring-boot-dependencies/pom.xml#L131-L151) (including _Spring_, and even _Spring_ core) can be overridden, and even other individual modules within other_Spring_ portfolio projects (e.g [Spring Security](https://github.com/spring-projects/spring-boot/blob/v1.4.0.M1/spring-boot-dependencies/pom.xml#L143-L145), or [Spring Social](https://github.com/spring-projects/spring-boot/blob/v1.4.0.M1/spring-boot-dependencies/pom.xml#L147-L150)).

Of course, these Spring projects may not include BOM for their collective modules.  But, what is similar is that not all SD modules in the SD portfolio (at a given version, e.g. [Hopper](https://github.com/spring-projects/spring-data-commons/wiki/Release-Train-Hopper#participating-modules)) are at the same version, individually.